Add a new skill to apply desired joint torque passed from the frankapy client

Ruthrash opened the pull request.

mohitsharma0690 commented on frankapy/franka_interface_common_definitions.py:
```diff
@@ -64,7 +65,7 @@ class FeedbackControllerType:
     PassThroughFeedbackController = _enum_auto('FeedbackControllerType')
     SetInternalImpedanceFeedbackController = _enum_auto('FeedbackControllerType')
-
+    TorqueFeedBackController = _enum_auto('FeedbackControllerType')
```
FeedBack should be Feedback, it's one word here.

mohitsharma0690 commented on frankapy/franka_arm.py:
```diff
@@ -164,13 +162,15 @@ def wait_for_franka_interface(self, timeout=None):
         timeout : :obj:`float`
             Timeout in seconds to wait for franka-interface.
         """
+
```
please remove these extra line additions.

mohitsharma0690 commented on bash_scripts/start_franka_ros_interface_on_control_pc.sh:
again, I don't think this file is needed.

mohitsharma0690 commented on bash_scripts/start_franka_gripper_on_control_pc.sh:
please remove all the extra changes in the bash file.

mohitsharma0690 reviewed (changes requested):
There seem to be too many files that shouldn't be added here. Can you please take a look.

Ruthrash commented:
Hey @mohitsharma0690! Apologies for getting so late to reply, I was travelling for the most part of the time after your change requests and I didn't have access to the franka robot. 

Also, so sorry for adding changes not relevant to this PR. I have removed them based on your comments. 

Thanks so much for your comments, I have incoporated the changes you have asked and have addressed some of your concerns. Please get back to me when you could. Thanks!

Ruthrash commented:
Also, I have an example joint_torque_control.py script that applies zero joint torque(gravity compensation) that can act as a test case for this feature. Do let me know if you think I should add it to this PR.
